css/styles.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+div.additional-info:empty {
```

Comment:
Use `hidden` tailwind utility

Reply:
h/t to @n1tranquilla for [docs](https://tailwindcss.com/docs/hover-focus-and-other-states#empty) on using class `empty:hidden`.